rllib/tests/backward_compat/test_backward_compat.py

```diff
@@ -66,11 +66,39 @@ def test_old_checkpoint_formats(self):
 
                     # Also test restoring a Policy from an algo checkpoint.
                     policies = Policy.from_checkpoint(path_to_checkpoint)
-                    assert "default_policy" in policies
+                    self.assertTrue("default_policy" in policies)
 
                 print(algo.train())
                 algo.stop()
 
+    def test_v1_policy_from_checkpoint(self):
```

Comment:
Just for my understanding: Any reason, why we shouldn't just do this test case in a loop?

B/c you just want to test Policy (not algo)? Python/pickle versions?

Reply:
you mean merge this test with the one above?
the thing is that I need to test DQN policies here. because that is the one we didn't finish migrating to PolicyV2.
it feels a special case compared to the other backward compatibility tests.
we can also remove this test once I move DQN, etc to PolicyV2.